http/org.wso2.carbon.transport.http.netty/src/main/resources/META-INF/services/org.wso2.carbon.messaging.ServerConnectorErrorHandler

```diff
@@ -1 +1 @@
-org.wso2.carbon.transport.http.netty.listener.WebSocketServerConnectorErrorHandler
\ No newline at end of file
+org.ballerinalang.services.dispatchers.websocket.WebSocketServerConnectorErrorHandler
```

Comment:
This is wrong, this entry should be added to a ballerina jar file, which has the implementation. You have indirectly put a dependency, where the implementation is not found in this jar.